docs/content/en/docs/concepts/executable-playbooks.md

```diff
@@ -0,0 +1,387 @@
+
+```json
+"target_definitions": {
+    "linux--b49069c2-0b69-4a46-8509-80196c4a9bf8": {
+      "type": "linux",
+      "name": "Target system",
+      "description": "System to execute commands on",
+      "address": {
+        "ipv4": [
+          "__target_ip__:value"
+        ]
+      }
+    }
+}
+```
+
+### Start and end steps
+
+Every CACAO playbook should start with a `start` step. From there, each step can define which step should be executed after the current step finishes. Depending on whether the current step has executed successfully, the next step is defined by the `on_success` or `on_failure` field. Alternatively an `on_completion` field can be defined, which identifies the next step regardless of the outcome of the current step.
+
+Example of a `start` step:
+
+```json
+"start--d6c44626-c9b6-426b-ad5d-3311bafaf068": {
+    "on_completion": "action--4b08af84-3741-48ca-8c92-df1557a87379",
+    "type": "start"
+}
+```
+
+According to the CACAO specification, every branch of steps should end in a (unique) `end` step. This is the only step type may not specify a next step:
+
+```json
+"end--60fc8d0c-3677-4363-8576-9ea9014f8c8e": {
+    "type": "end"
+}
+```
+
+### If-condition, while-condition, and parallel steps
+
+An `if-condition` step allows executing different branches depending on a specified condition. The step must specify an `on_true` field, which references the start of a branch of steps that should be executed if the condition evaluates to true. Optionally, the `if-condition` step can define an `on_false` field that defines an alternative branch that is executed if the condition evaluates to false. In each case, the specified branch keeps executing until it encounters an `end` step.
+
+The `condition` field contains a string that specifies a [STIX Pattern](https://docs.oasis-open.org/cti/stix/v2.0/stix-v2.0-part5-stix-patterning.html). Currently, SOARCA only supports a very small subsection of the STIX Patterning specification. We support string based equality (`'a' = 'a'`) and inequality (`'a' != 'b'`) comparison. Example:
+
+```json
+"if-condition--4b95eaa4-944a-4a9d-88d4-1374a70dbacd": {
+    "name": "If it is not new years",
+    "description": "Checks if it is 01-01-2025",
+    "on_completion": "end--db937fc8-3a42-41cc-b828-ec2db212f425",
+    "type": "if-condition",
+    "condition": "__soarca_ssh_result__:value != '01-01-2025'",
+    "on_true": "action--7fe08053-3685-4d8c-bc0a-40efce75113e"
+}
+```
+
+SOARCA supports variable interpolation, which means that variables can be used inside the `condition` field, as seen in the example above.
+
+Similarly, CACAO specifies `while-condition` steps, whose `on_true` branch will be repeatedly executed until the condition evaluates to false. The execution limit is currently set to 10. If the condition is still true after 10 iterations, the step ends in a failure.
```

Comment:
This can be unlimited as long as one can break, the execution limit per iteration should be set by the timeout

Reply:
@lucamrgs would a step-level iteration maximum be a good extension to the standard?

I added the limit for testing locally, but we can just drop it from the docs.